hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
@Swizz attribute typing is a cool feature that lets you spot a lot of errors at compile time, so we should attempt to cover it. We don't have to implement it from scratch, we can easily adapt `react` or `preact` (their typings are MIT licensed). Custom attributes are not a problem: they do appear in custom elements which in turn are extensions of native HTML elements; their typings should be automatic if the component is written the right way.

The only problem I see is with events when an action is passed as event handler, e.g.
```js
<button onclick={actions.some}></button>
```
but I think that can be solved by something like `EventHanlder | Action`